src/main/kotlin/org/olafneumann/regex/generator/regex/PatternHelper.kt

```diff
@@ -1,5 +1,3 @@
 package org.olafneumann.regex.generator.regex
 
-//object PatternHelper {
-fun String.escapeForRegex() = this.replace(RegexCache.get("([.\\\\^$\\[{}()*?+])"), "\\\\$1")
-//}
+fun String.escapeForRegex() = this.replace(RegexCache.get("([|.\\\\^$\\[{}()*?+])"), "\\\\$1")
```

Comment:
Hint for review: I added the pipe symbol in this regex (in the RegexCache#get call)